Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.1.2
+FROM ruby:3.3.4
```

Comment:
Could you please confirm that the version is available from Docker Hub?

Reply:
Yes, it is available